_data/glossary.yml

```diff
@@ -1,9 +1,79 @@
+- keyword: Health Information 
+  definition: General information provided about health and care with the absence of triage or symptoms
+- keyword: HL7
+  expansion: Health Level 7
+  definition: HL7.org is an international organisation responsible for the creation and maintenance of information and interoperability standards for healthcare.
+- keyword: Health Outcome 
+  definition: Refers to impact of over triage or under triage to patient well being, as well as the impact on patients when they follow advice given or the impact of not following the advice e.g. mortality, hospitalisation, or re-presentation of symptoms. Health outcomes require longer term monitoring of the patient than immediate clinical outcomes ; the health outcome is not always immediately obvious
+- keyword: HTTP  
+  expansion: Hyper Text Transfer protocol
+  definition: Hypertext Transfer Protocol is an application protocol used for data communication in the World Wide Web. It provides a standard for web browsers that facilitates users to exchange information over the internet.
+- keyword: ITK
+  expansion: Interoperability Tool Kit
+  definition: A collection of interoperability standards and tools produced by NHS Digital to support interoperability in healthcare.
+- keyword: IUC CAS 
+  expansion: Integrated Urgent Care Clinical Assessment Service
+  definition: Physical and/or virtual hub providing clinical advice to patients contacting the 111 or 999 services, as well as providing clinical support to clinicians (particularly ambulance staff such as paramedics and emergency technicians) to ensure that no decision is made in isolation. Also known as Care Advice Service or Clinical Assessment Service, previously known as a Clinical Navigation Hub.
+- keyword: Patient 
+  definition: Generic term used in Commissioning Standards for citizens / public using the service.
+- keyword: PMPR  
+  expansion: Predictive Modelling of Patient Risk
+  definition: Using historical data to identify the patient-level variables that predict over-triage and under-triage for different Pathways or symptom groups. These patient-level variables can then be used to identify future service users who are at high risk of over- and under-triage.
+- keyword: POC 
+  expansion: Proof of Concept
+  definition: A proof of concept is a demonstration, the purpose of which is to verify that certain concepts or theories have the potential for real-world application. POC is therefore a prototype that is designed to determine feasibility, but does not represent deliverables.
+- keyword: Provider 
+  definition: Entity contracted by a commissioner to provide a service
+- keyword: SNOMED CT 
+  expansion: Systemised Nomenclature of Medicine (Clinical Terms). 
+  definition: A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world.
```

Comment:
Suggest adding a mention that it is the mandated official clinical terminology for the NHS.

Reply:
Amended:
A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world. SNOMED CT is to be used as the single terminology in all care settings in England.